pkg/console/subresource/configmap/configmap.go

```diff
@@ -100,8 +100,8 @@ func DefaultConfigMap(
 		ReleaseVersion().
 		NodeArchitectures(nodeArchitectures).
 		NodeOperatingSystems(nodeOperatingSystems).
-		OAuthClientConfig(authConfig, oidcConfig).
-		AuthServerCA(authConfig, oidcConfig).
+		OAuthClientConfig(authConfig).
+		AuthServerCA(authConfig, authServerCAConfig).
```

Comment:
I'm thinking we may want to merge these functions into one that deals with configuring the auth options

Reply:
Good idea :)